Week03/Implement Trie.cpp

```diff
@@ -0,0 +1,60 @@
+//TC: O(n) for each function call where n is the length of the string
+//MC: O(t) where t is the total number of TrieNodes created in the Trie
```

Comment:
That's vague. Just say, O(M) where M is the total number of characters in all the words inserted. Straight forward, on point.